firestore/google/cloud/firestore_v1beta1/_helpers.py

```diff
@@ -515,6 +517,31 @@ def encode_dict(values_dict):
     }
 
 
+def extract_field_paths(actual_data):
```

Comment:
`acutal_data` is `document_data` in the Node/Java SDK.

Reply:
Done.